### PR TITLE
feat(query): add 'App Service Client Certificates Disabled' for Terraform

### PR DESCRIPTION
**Pull request by tiagocruz20**
**Proposed Changes**
- added a query to check if client certificates enabled is true for azure app service resource

I submit this contribution under the Apache-2.0 license.

### CODE REVIEW

**Comment by rogeriopeixotocx** [comment]
Pending **Severity** and **Category** review

File: assets/queries/terraform/azure/azure_app_service_client_certificate_disabled/metadata.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "id": "a81573f9-3691-4d83-88a0-7d4af63e17a3",
+  "queryName": "Azure App Service Client Certificate Disabled",
+  "severity": "",
+  "category": "",
```
Comment (rogeriopeixotocx) [inline assets/queries/terraform/azure/azure_app_service_client_certificate_disabled/metadata.json:5]:
```suggestion
  "category": "Insecure Configurations",
```

**Review by rogeriopeixotocx — CHANGES_REQUESTED** [review]
format rego query either using `opa fmt` [link](https://github.com/open-policy-agent/opa/blob/c6344d05256c2ceb6ad4db3feb88922d44649862/cmd/fmt.go) or by using vscode OPA extension and enabling format on save on settings

File: payload.json [hunk 2]
```diff
@@ -0,0 +1,84 @@
+{
```
Comment (rogeriopeixotocx) [inline payload.json:1]: Remove this file.

**Review by rogeriopeixotocx — CHANGES_REQUESTED** [review]

File: assets/queries/terraform/azure/azure_app_service_client_certificate_disabled/query.rego [hunk 3]
```diff
@@ -0,0 +1,33 @@
+package Cx
```
Comment (rogeriopeixotocx) [inline assets/queries/terraform/azure/azure_app_service_client_certificate_disabled/query.rego:1]: Format this file using opa fmt or vscode extension for OPA

File: assets/queries/terraform/azure/azure_app_service_client_certificate_disabled/test/negative.tf [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+  resource_group_name = azurerm_resource_group.example.name
+  app_service_plan_id = azurerm_app_service_plan.example.id
+
+  site_config {
+    dotnet_framework_version = "v4.0"
+    scm_type                 = "LocalGit"
+  }
+
+  app_settings = {
+    "SOME_KEY" = "some-value"
```
Comment (rogeriopeixotocx) [inline assets/queries/terraform/azure/azure_app_service_client_certificate_disabled/test/negative.tf:13]:
```suggestion
    SOME_KEY = "some-value"
```

File: assets/queries/terraform/azure/azure_app_service_client_certificate_disabled/test/positive1.tf [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+  resource_group_name = azurerm_resource_group.example.name
+  app_service_plan_id = azurerm_app_service_plan.example.id
+
+  site_config {
+    dotnet_framework_version = "v4.0"
+    scm_type                 = "LocalGit"
+  }
+
+  app_settings = {
+    "SOME_KEY" = "some-value"
```
Comment (rogeriopeixotocx) [inline assets/queries/terraform/azure/azure_app_service_client_certificate_disabled/test/positive1.tf:13]:
```suggestion
    SOME_KEY = "some-value"
```

File: assets/queries/terraform/azure/azure_app_service_client_certificate_disabled/test/positive2.tf [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+  resource_group_name = azurerm_resource_group.example.name
+  app_service_plan_id = azurerm_app_service_plan.example.id
+
+  site_config {
+    dotnet_framework_version = "v4.0"
+    scm_type                 = "LocalGit"
+  }
+
+  app_settings = {
+    "SOME_KEY" = "some-value"
```
Comment (rogeriopeixotocx) [inline assets/queries/terraform/azure/azure_app_service_client_certificate_disabled/test/positive2.tf:13]:
```suggestion
    SOME_KEY = "some-value"
```

File: assets/queries/terraform/azure/azure_app_service_client_certificate_disabled/test/positive_expected_result.json [hunk 7]
```diff
@@ -0,0 +1,14 @@
+[
+  {
+    "queryName": "Azure App Service Client Certificate Disabled",
+    "severity": "",
```
Comment (rogeriopeixotocx) [inline assets/queries/terraform/azure/azure_app_service_client_certificate_disabled/test/positive_expected_result.json:4]:
```suggestion
    "severity": "HIGH",
```

File: assets/queries/terraform/azure/azure_app_service_client_certificate_disabled/test/positive_expected_result.json [hunk 8]
```diff
@@ -0,0 +1,14 @@
...
+    "line": 1,
+    "fileName": "positive1.tf"
+  },
+  {
+    "queryName": "Azure App Service Client Certificate Disabled",
+    "severity": "",
```
Comment (rogeriopeixotocx) [inline assets/queries/terraform/azure/azure_app_service_client_certificate_disabled/test/positive_expected_result.json:10]:
```suggestion
    "severity": "HIGH",
```

**Review by rogeriopeixotocx — CHANGES_REQUESTED** [review]

**Comment by rogeriopeixotocx** [comment]
@tiagocruz20 please address the requested changes

**Review by rafaela-soares — APPROVED** [review]
LGTM

**Review by rogeriopeixotocx — APPROVED** [review]
LGTM

**Pull request merged**